pymdp/planning/mcts.py

```diff
@@ -1,12 +1,19 @@
 from functools import partial
 from jax import vmap, nn, random as jr, tree_util as jtu, lax
-from pymdp.jax.control import compute_expected_state, compute_expected_obs, compute_info_gain, compute_expected_utility
+from pymdp.control import (
```

Comment:
To avoid these formatting diffs I suggest before merging to v1.0.0 alpha / master, we add in some kind of CI workflow step with `black` and a pre-commit hook -- see https://github.com/infer-actively/pymdp/issues/183